specs/logging_changes_spec.adoc

```diff
@@ -0,0 +1,171 @@
+// vim: tw=avored-markdown/preview.ht79
+
+# Changes in tendrl modules after introduction of new logging module in common
+
+Since a new logging module is being introduced in Tendrl-common(https://github.com/Tendrl/specifications/pull/94), after its implementation
+there will be requirement of changes in all the modules of Tendrl where logging is used.
+This file proposes all those changes and how they will be made.
+
+
+## Problem description
+
+Many modules in Tendrl have a functionality of logging different types of situations/use cases such as logging (info, errors, warning etc.).
+Till now these were implemented by importing the Python standard library's logging module then calling logger functions to handle the logging.
+After implementation of new logging module there will be requirement of changes in all the modules of Tendrl where logging is used.
+
+#### Use Cases
+
+There will be 3 classes(use cases) that will be provided by the common logging module:
```

Comment:
I feel use cases are logging, alerting and in flight operation steps and below classes are defined for that. We should not directly talk about classes. Rather we should say "three use cases which would be tackled are logging, alerting and in flight operations".